friends/src/components/Form.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import './friends.css';
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      age: "",
+      email: ""
+    };
+  }
+
+  onChangeHandler = event => {
+    if(event.target.type === "number"){
+      this.setState({
+        [event.target.name]: Number(event.target.value)
+      });
+    }else{
+      this.setState({
+        [event.target.name]: event.target.value
```

Comment:
You could make this shorter and easier to read with some destructuring and by moving the check out to happen before you set state:
```javascript
onChangeHandler = ({ target }) => {
  let { name, type, value } = target;
  if (type == 'number') {
    value = Number(value);
  }
  this.setState({
    [name]: value
  });
};
```